localization/i18n/es/OrcaSlicer_es.po

```diff
@@ -2930,7 +2937,7 @@ msgid "Nozzle temperature"
 msgstr "Temperatura de la boquilla"
 
 msgid "Bed Temperature"
-msgstr "Temperatura de Cama"
+msgstr "Temperatura de cama"
```

Comment:
should stay as Cama since its Bed Temperature.

Reply:
I have doubts about these translations. There are a lot of capitalised strings in English, sometimes they seem to be correct and sometimes not. I'm going to correct it as you say anyway